dockerfiles/init/modules/openshift/files/scripts/multi-user/postgres/build-config.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: v1
+kind: BuildConfig
+metadata:
+  name: postgres-for-che
+spec:
+  nodeSelector: null
+  output:
+    to:
+      kind: ImageStreamTag
+      name: 'postgres:latest'
+  postCommit: {}
+  resources: {}
+  runPolicy: Serial
+  source:
+    images:
+      - from:
+          kind: ImageStreamTag
+          name: 'che-init:latest'
+        paths:
+          - destinationDir: ./
+            sourcePath: /etc/puppet/modules/postgres/templates/init-che-user.sh.erb
+          - destinationDir: .s2i/bin/
+            sourcePath: /files/s2i/postgres/assemble
+          - destinationDir: .s2i/bin/
+            sourcePath: /files/s2i/postgres/run
+    type: Image
+  strategy:
+    sourceStrategy:
+      from:
+        kind: ImageStreamTag
+        name: 'postgres-source:latest'
+    type: Source
+  triggers:
+  - type: "ConfigChange"
+  - type: "ImageChange" 
+    imageChange: {}
+  - type: "ImageChange" 
+    imageChange:
+      from:
+        kind: "ImageStreamTag"
+        name: "che-init:latest"
```

Comment:
is it OK to use always `latest` tag here? I have no idea how these yamls will be used but I think that could be a problem to use always latest.

Reply:
yes, because the `che-init:latest` image stream tag is itself generated from the configured  docker `che-init` docker image (cf the IMAGE_INIT environment variable)